WcaOnRails/app/webpacker/components/Media/MediaTable.jsx

```diff
@@ -0,0 +1,119 @@
+import _ from 'lodash'
+import React, { useState, useEffect } from 'react';
+import { Table } from "semantic-ui-react";
+import { countries } from '../../lib/wca-data.js.erb';
+import { competitionUrl } from '../../lib/requests/routes.js.erb';
+import moment from 'moment';
+
+function dateRangeBetween(fromDate, toDate) {
```

Comment:
Would you want to create a file in `utils/` for formatting date strings and start putting code like this there?

Reply:
I have moved this to libs/helper as its already created used